website/pages/en/index.js

```diff
@@ -14,41 +14,41 @@ const siteConfig = require(process.cwd() + "/siteConfig.js");
 const translate = require("../../server/translate.js").translate;
 
 const pre = "```";
-const typeAnnotationExample= `${pre}hack
-<?hh
-class MyClass {
-  const int MyConst = 0;
-  private string $x = '';
-  public function increment(int $x): int {
-    $y = $x + 1;
-    return $y;
+const asyncExample= `${pre}hack
+  async function hello_world(): Awaitable<string> {
+    return "Hello World"; // simple, trivial example
+  }
+
+  <<__EntryPoint>>
+  async function call_hello_world(): Awaitable<void> {
+
+  $aw = hello_world();
+  $result = await $aw; // Suspends until $aw completes
```

Comment:
Thinking about how we describe async, I suppose the important thing is that you can wait for multiple things at once.

I can't see a good way of writing a small self-contained example for this. How about something like this?

```
<<__EntryPoint>>
async function my_example(): Awaitable<void> {
  $user_ids = vec[1, 2, 3];

  // This will initiate all the database requests together, so we
  // spend less time waiting.
  $user_names = await Vec\map_async(
    $user_ids,
    async ($id) ==> await fetch_user_name($id),
  );
  // Execution continues once all the requests have completed.

  echo Str\join($user_names, ", ");
}

async function fetch_user_name(int $_): Awaitable<string> {
  // In practice this would be a database request.
  return "";
}
```

We could even drop the `fetch_user_name` function if you think it's distracting.

Reply:
Had the same thought here -- Just couldn't find a way to get it all neatly self-contained so I used a trivial example.